README.md

```diff
@@ -1,17 +1,61 @@
-### License
+SonarQube [![Build Status](https://travis-ci.org/SonarSource/sonarqube.svg?branch=master)](https://travis-ci.org/SonarSource/sonarqube) [![Quality Gate](https://next.sonarqube.com/sonarqube/api/badges/gate?key=org.sonarsource.sonarqube%3Asonarqube)](https://next.sonarqube.com/sonarqube/dashboard?id=org.sonarsource.sonarqube%3Asonarqube) [![Coverage](https://next.sonarqube.com/sonarqube/api/badges/measure?key=org.sonarsource.sonarqube%3Asonarqube&metric=coverage)](https://next.sonarqube.com/sonarqube/component_measures/domain/Coverage?id=org.sonarsource.sonarqube%3Asonarqube)
+=========
 
-Copyright 2008-2017 SonarSource.
+Continuous Inspection
+---------------------
+SonarQube provides the capability to not only show health of an application but also to highlight issues newly introduced. With a Quality Gate in place, you can fix the leak and therefore improve code quality systematically.
 
-Licensed under the [GNU Lesser General Public License, Version 3.0](http://www.gnu.org/licenses/lgpl.txt)
+Links
+-----
 
-### Build status
+* [Website](https://www.sonarqube.org)
+* [Download](https://www.sonarqube.org/downloads/)
+* [Documentation](https://docs.sonarqube.org)
+* [Twitter](https://twitter.com/SonarQube)
+* [SonarSource](https://www.sonarsource.com), editor of SonarQube
+* [Issue tracking](https://jira.sonarsource.com/browse/SONAR/), read-only. Only SonarSourcers can create tickets.
+* [Demo](https://next.sonarqube.com/sonarqube/dashboard?id=org.sonarsource.sonarqube%3Asonarqube) of the next version to be released
 
-[![Build Status](https://travis-ci.org/SonarSource/sonarqube.svg?branch=master)](https://travis-ci.org/SonarSource/sonarqube)
+Have Question or Feedback?
+--------------------------
 
-### Links
+To provide feedback (request a feature, report a bug etc.) use the [SonarQube Google Group](https://groups.google.com/forum/#!forum/sonarqube). Please do not forget to specify the version.
```

Comment:
While you are at providing advice, what about courtises and such? Maybe a link of those guide lines are available somewhere

Reply:
I suggest to summon @ganncamp on this: is there any guideline to contribute to the Google Group somewhere, @ganncamp ?

Reply:
@sns-seb it's in the [group header](https://groups.google.com/forum/?pli=1#!forum/sonarqube)